integration/test/performance_tracker/transmitter/transmitter.go

```diff
@@ -0,0 +1,236 @@
+package transmitter
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/session"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/dynamodbiface"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+const METRIC_PERIOD= 300.0 // this const is in seconds , 5 mins
```

Comment:
Why would we want to initiate this variable to `float` and in later time we convert the [numDataPoints](https://github.com/aws/amazon-cloudwatch-agent/pull/496/files#diff-c3d3ee8804154aa85568c8778e4b9e58dcae04e6a2d03c416a727c4a3f4457fcR214) to float again?

Reply:
Maybe a different question, do we need to do floating point division and then round by casting to an int, or can we change it around?

The issue is around using the number of data points to calculate the average. You could just keep numDataPoints as an int, and then cast when calculating the average, e.g.:
```go
avg := sum / float64(numDataPoints)
```
which leaves numDataPoints as an int. Then we don't need to do the casting when doing METRIC_PERIOD / numDataPoints